e2e_test.ts

```diff
@@ -881,6 +881,9 @@ Deno.test("[e2e] GET /account/upgrade", async (test) => {
   await createUser(user);
 
   await test.step("serves internal server error response if the `STRIPE_PREMIUM_PLAN_PRICE_ID` environment variable is not set", async () => {
+    // Suppress the error message thrown by the handler
+    const stubbedError = stub(console, "error");
```

Comment:
```suggestion
    using stubbedError = stub(console, "error");
```
Removes the need for `stubbedError.restore();` below.

Reply:
TS doesn't like it
<img width="1155" alt="Screenshot 2024-10-31 at 20 01 45" src="https://github.com/user-attachments/assets/041055c9-21d4-460c-926c-932e7e595ba3">

Reply:
Ah. Ignore then :)